### PR TITLE
Get token from cookies if not in headers or params

### PR DESCRIPTION
**Pull request by padwasabimasala**
Checks for auth token in cookies after headers and params

@jaywagnon please review

### CODE REVIEW

File: lib/rack/auth/oc_tanner/token.rb [hunk 1]
```diff
@@ -43,14 +43,18 @@ def token
...
+      token = ENV['OCTANNER_AUTH_TOKEN'] = token_from_headers || token_from_params || token_from_cookies
     end
   end
 
   def request
     request ||= Rack::Request.new @env  # todo is 'request' a local variable here? why?
   end
 
+  def token_from_cookies
+    return request.cookies['_access_token']
```
Comment (jaywagnon) [inline lib/rack/auth/oc_tanner/token.rb]: Can we name it 'bearer_token' to match the other inputs? Is the leading underscore significant?

Comment (jaywagnon) [inline lib/rack/auth/oc_tanner/token.rb]: Ah, heh...they all start with a leading underscore. Shows how much I work with cookies. ;P

Comment (padwasabimasala) [inline lib/rack/auth/oc_tanner/token.rb]: The name '_access_token' has been in place for a very long time, and it is what the other front end apps use, so we're stuck with it for now.

Comment (jaywagnon) [inline lib/rack/auth/oc_tanner/token.rb]: "access_token" is deprecated in other parts of the code.  Could you put in an OR condition with "bearer_token" so there is a migration path, please?

**Comment by padwasabimasala** [comment]
how bout dat?

**Comment by jaywagnon** [comment]
Beauty!  Thank ya!

**Pull request merged**